src/bundle/Resources/public/js/scripts/button.translation.edit.js

```diff
@@ -52,14 +51,38 @@
             }
 
             this.setPosition();
-
             this.backdrop.show();
-            doc.addEventListener('click', this.hideTranslationsList, false);
+            this.closeBtn.addEventListener('click', this.hideExtraActionPanel, false);
+
             ibexa.helpers.tooltips.hideAll();
         }
 
+        setActiveLanguage(event) {
+            const { contentId, languageCode } = event.currentTarget.dataset;
+
+            this.confirmBtn.dataset.contentId = contentId;
+            this.confirmBtn.dataset.languageCode = languageCode;
+            this.confirmBtn.disabled = false;
+
+            this.languagesBtns.forEach((btn) => btn.classList.remove('ibexa-btn--active'));
```

Comment:
considering you can have only one active language, wouldn't it be easier to get only that one and remove this class?